charts/linkerd2-multicluster-remote-setup/templates/service-mirror-rbac.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Should we have a version of this that is namespace specific? I'm wondering if some folks will want to pass a list of namespaces to be mirrored at install time and have this be a RoleBinding instead.